src/modules/reporting/components/reporting-report/reporting-report.jsx

```diff
@@ -103,8 +110,26 @@ export default class ReportingReport extends Component {
             history={p.history}
             cardStyle="single-card"
             buttonText="View"
+            showAdditionalDetailsToggle={s.showToggleDetails}
```

Comment:
Could we move this out of the MarketPreview component and style it to look like it belongs there?

Reply:
Definitely missed this, will correct.

Reply:
Based on the Mocks this is the best component architecture I could come up with. Hopefully the Mocks will change to allow for a better component layout. If anyone knows of a better way to architect this I'll need direction.